gsms.py

```diff
@@ -53,15 +53,83 @@
 
 # standard imports
 import argparse
+import ctypes
+from ctypes import wintypes
 import json
 import os
+import re
 import shutil
 import time
+import uuid
 
 # lib imports
 import pylnk3
 
 
+# Code from here and modified to work in this project
+# https://gist.github.com/mkropat/7550097
+class GUID(ctypes.Structure):
+    """
+    Class to build a GUID compliant object for use in WIndows libraries
+
+    Parameters
+    -------
```

Comment:
- The `-` line should be equal length to the section name.

Reply:
Done for all

Reply:
> Done for all

Except this one?

Reply:
im unsure how I missed it... I even added more docs right below